packages/logger/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@plane/logger",
+  "version": "1.0.0",
+  "description": "Logger shared across multiple apps internally",
+  "private": true,
+  "main": "./dist/index.js",
+  "type":"module",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "build": "tsup ./src/index.ts --format esm,cjs --dts --external react --minify",
```

Comment:
donot create a build for this package. You can directly import them and use it. Let the application that is using it transpile these packages.